modules/gcloud/src/test/java/org/testcontainers/containers/DatastoreEmulatorContainerTest.java

```diff
@@ -40,6 +40,20 @@ public void testSimple() {
 
         assertThat(datastore.get(key).getString("description")).isEqualTo("my description");
     }
+
     // }
 
+    @Test
+    public void testWithFlags() {
```

Comment:
Let's add one more test with multiple flags (see my comment above, good way to check for this).

Reply:
done